grammar.js

```diff
@@ -491,9 +519,16 @@ module.exports = grammar({
     subroutine_call: $ => seq(
       caseInsensitive('call'),
       $._name,
+      optional($.cuda_kernel_argument_list),
       optional($.argument_list)
     ),
 
+    cuda_kernel_argument_list: $ => seq(
+      '<<<',
+      commaSep1($._expression),
```

Comment:
Might want to double check if these kernel expressions can span more than one line and if so add a test case for it.

Reply:
Added.